.github/publish-website.yml

```diff
@@ -0,0 +1,99 @@
+name: Publish website
```

Comment:
This doesn't seem to have run (I don't know why), but it wouldn't work anyway: there are two jobs here that both push to the gh-pages branch, one overwriting the other. I did some additional work on solid-client to package them both together into a single deployment: https://github.com/inrupt/solid-client-js/commit/2f27f14fb172628960e1b6f976fd23e0284d33be